openpype/hosts/photoshop/api/ws_stub.py

```diff
@@ -360,6 +360,36 @@ def select_layers(self, layers):
             )
         )
 
+    def get_activeDocument_format_resolution(self):
+        """Return the width and height in pixel of the active doc
+
+        Returns(dict):
+            {"width": int, "height": int}
+        """
+        res = self.websocketserver.call(
+            self.client.call('Photoshop.get_activeDocument_format_resolution')
+        )
+
+        return json.loads(res)
```

Comment:
I don't think `json.loads(res)` witll work without crashing if `res` is null and not `{}`

Reply:
Corrigé avec un if